Add header link items mobile

helloimela opened the pull request:
**Describe pull-request**</br>
Added header item links in mobile view
Add fixed view with 100% height in mobile view


**Additional context**<br/>
Related PR  https://github.com/scania/corporate-ui-dev/pull/174

Commit:
Add license information

Commit:
Refactored header and navigation

- Add scania symbol in navigation
- Add manual sticky nav for IE
- Remove stuck styles from header

Commit:
Merge branch 'master' into improvement/separate_nav_header

Commit:
Added style to avoid flicker in sticky nav

- Added custom margin and padding so it does not flicker on stickynav on IE fixed navigation
- Added border-bottom on navigation
- Remove overflow hidden from navigation mobile

Commit:
Merge branch 'master' into improvement/separate_nav_header

Commit:
Added styling for mobile menu toggle menu

Commit:
Reset margin navigation on mobile view

Commit:
Add header item links in mobile view

Commit:
Add tgz for preview in corporate-ui-dev aws branch

Commit:
Added fix to header & nav height bug

- Navigation height was 100% cause it to cover the content, remove uneccessary height

awinny commented on src/styles/c-navigation.scss:
```diff
@@ -78,8 +84,11 @@ ul {
-  transition: margin 250ms ease;
+  transition: all 250ms ease;
   order: -1; /* This is needed for ie - due to it beeing render after slots */
+  height: calc(75vh - 64px);
```
Would be nice with a comment describing what the this calculation is about

helloimela replied:
comment has been added in the new changes

awinny commented on src/styles/c-header.scss:
```diff
@@ -182,28 +241,22 @@ $link-color: #747472;
-      -webkit-box-shadow: 0px 17px 15px -15px rgba(0, 0, 0, 0.15);
-      box-shadow: 0px 17px 15px -15px rgba(0, 0, 0, 0.15);
+// IE special handling
+@media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
```
Try to set the position static as a general rule for all slotted a nodes in all browsers

helloimela replied:
Ok, I change some properties so we don't need this rule anymore

awinny commented on src/styles/c-header.scss:
```diff
@@ -137,17 +179,16 @@ $link-color: #747472;
-  ::slotted(c-navigation){
-    background-color: transparent;
-    border-top: 1px solid #e2e2e2;
+    * {
```
Do we really need this?

helloimela replied:
Nope, I will remove this

awinny reviewed (changes requested):
See the line comments

Commit:
Fixes for review

- Remove IE specific rule
- added comment to make things clearer
- remove unnecessary transition properties

Commit:
Remove tar gz file that was used for demo

awinny reviewed (approved):
Seems good 👍
